AutoLayoutHelper/UIView+AutoLayoutHelper.swift

```diff
@@ -300,14 +300,21 @@ extension UIView {
     @discardableResult func addHeightConstraint(toView view: UIView?, relation: NSLayoutRelation = .equal, constant: CGFloat = 0.0) -> NSLayoutConstraint {
         
         let constraint: NSLayoutConstraint = self.createConstraint(attribute: .height, toView: view, attribute: .height, relation: relation, constant: constant)
-        self.superview?.addConstraint(constraint)
+        self.addConstraintToSuperview(constraint)
         
         return constraint
     }
     
 
     // MARK: - Private
     
+    /// Adds an NSLayoutConstraint to the superview
+    fileprivate func addConstraintToSuperview(_ constraint: NSLayoutConstraint) {
+        
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Looking at discussion over at #20 I see we're opting to remove it so ignore this comment.

We should also look at Swift Lint as a way of checking style rules but not for this merge request :)